tests/testsuite/aws.rs

```diff
@@ -4,12 +4,11 @@ use std::io::{self, Write};
 use ansi_term::Color;
```

Comment:
I think it would be best if we added tests that explicitly tested the `displayed_items = "all"` configuration, in case we ever change the default behavior. 🙂

Reply:
Fair point, I've added the tests.